server/src/database/mod.rs

```diff
@@ -0,0 +1,35 @@
+use postgres::{Connection, TlsMode, Error};
+use std::borrow::Borrow;
+use std::{thread, time};
+use super::utils::get_env_variable;
+
+/// Try n-times to get a new connection to a Postgres database.
+fn try_connection(params: &String, tries: u8) -> Result<Connection, Error> {
+    match Connection::connect((*params).borrow(), TlsMode::None) {
```

Comment:
Does `params.borrow()` work? Instead of `(*params).borrow()`. I'm not sure it works but it would be more readable imo.